main.go

```diff
@@ -3,75 +3,121 @@ package main
 import (
 	"flag"
 	"log"
-	"sync"
+
+	"github.com/Regner/albionmarket-client/utils"
 
 	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
 	"github.com/google/gopacket/pcap"
-	"github.com/regner/albionmarket-client/assemblers"
-	"github.com/regner/albionmarket-client/utils"
+	"github.com/hmadison/ao_spectator/photon"
+	"github.com/hmadison/ao_spectator/albion"
+	"github.com/mitchellh/mapstructure"
 )
 
+type RequestBuyOrders struct {
```

Comment:
It's my understanding that that package wasn't going to have albion specific stuff, I'm not sure why it's in there at all to be honest. Maybe I misunderstood.